docs/framework/network-programming/network-programming-samples.md

```diff
@@ -10,40 +10,40 @@ This section contains descriptions and links to downloadable network programming
 > All samples are available in the [.NET Framework SDK version 2.0 Samples Download](https://www.microsoft.com/download/confirmation.aspx?id=22181), which was released for the .NET Framework 2.0 and may be dated.
 
 ## In This Section  
- [Download Progress Indicator Technology Sample](https://msdn.microsoft.com/library/t8w6294a(v=vs.85).aspx)  
+ [Download Progress Indicator Technology Sample](https://docs.microsoft.com/previous-versions/dotnet/netframework-3.0/t8w6294a(v=vs.85))  
```

Comment:
It seems that the latest version of this would be under the vs.90. Can you review the links on this page to see if you can find them under the vs.90?

```suggestion
 [Download Progress Indicator Technology Sample](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2008/t8w6294a(v=vs.90))  
```

Reply:
@mairaw I found two differences in the links inside those pages:

1. In the vs.85 version, the **System.Net** link in the See Also section goes to a current (non-previous-version) page on docs.microsoft.com containing long lists of classes/interfaces/enums/delegates, while the vs.90 version has the link going to an MSDN page (https://msdn.microsoft.com/en-us/library/btdf6a7e) containing just one class and one interface.

2. The **How to: Set Sample Settings** link goes to a more verbose topic from the vs.85 page than from the vs.90 page, though not as much as with **System.Net**. The article linked to from the vs.90 page basically says to follow what's described in VS2005/.NET2.0 and later versions. The explicit procedure in the linked articles are identical, so I think the pages are equivalent.

Also, the vs.85 and vs.90 pages themselves are the same other than some formatting differences, so I'd recommend sticking with the vs.85 page. (The previous-versions docset for vs.85 refers to dotnet as opposed to visualstudio for vs.90, so I assumed you'd prefer a dotnet URL if all other things were equal.)

Reply:
Ok sounds good. Thanks for your analysis @DennisLee-DennisLee!